WordPressAuthenticator/Authenticator/WordPressAuthenticator.swift

```diff
@@ -90,10 +90,6 @@ import AuthenticationServices
                                   unifiedStyle: WordPressAuthenticatorUnifiedStyle?,
                                   displayImages: WordPressAuthenticatorDisplayImages = .defaultImages,
                                   displayStrings: WordPressAuthenticatorDisplayStrings = .defaultStrings) {
-        guard privateInstance == nil else {
```

Comment:
During testing, it's far easier to allow this to be called multiple times than to crash when it is.  And honestly during normal usage of the App, we haven't seen this crashing, and I'm not sure we want it to.